src/mlpack/methods/ann/layer/advantage_loss.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file advantage_loss.hpp
+ * @author Rohan Raj
+ *
+ * Definition of the Advantage error performance function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_ADVANTAGE_ERROR_HPP
+#define MLPACK_METHODS_ANN_LAYER_ADVANTAGE_ERROR_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The Advantage error performance function measures the network's
+ * performance on the basis of advantages.
+ *
+ * Reference : https://karpathy.github.io/2016/05/31/rl/
```

Comment:
Do you think we can find a paper, that describes the method we can cite here?

Reply:
@zoq ,The paper we should cite here is https://arxiv.org/pdf/1506.02438.pdf . The policy gradient method or generalised advantage estimation is what we are trying to solve here . I would like to know your views.

Reply:
Sounds good.